libgalois/include/katana/analytics/ClusteringImplementationBase.h

```diff
@@ -214,20 +215,20 @@ struct ClusteringImplementationBase {
    * without swapping the cluster assignment.
    */
   void MaxModularityWithoutSwaps(
-      std::map<uint64_t, uint64_t>& cluster_local_map,
+      std::map<CommunityIdTy, CommunityIdTy>& cluster_local_map,
```

Comment:
not sure is this makes sense. it should be std::map<CommunityIdTy, uint64_t>, no?

Reply:
Is it mapping the current -> the next cluster?

Reply:
I think it was mapping the number of neighbors in each of the cluster/community

Reply:
I see. Then I guess uint32_t would be right since we generally use uint32_t for the number of nodes, and uint64_t for the number of edges.

Reply:
you are right. uint32_t is right then

Reply:
BTW, I thought about your suggestion again and it would be fine.
I consider `CommunityIdTy` as the number of nodes of the coarsened graph. I just use this type due to readability.

Reply:
yeah, both of them are fine for now. since communtiyIdTy will always be an unsigned int.